Client/game_sa/CObjectSA.h

```diff
@@ -155,6 +155,9 @@ class CObjectSA : public virtual CObject, public virtual CPhysicalSA
     CVector* GetScale();
     void     ResetScale();
 
+    float GetRopeHeight();
```

Comment:
The method GetRopeHeight doesn't change anything in the class, right? The similar methods should be marked as **const**.